dateutil/parser/_parser.py

```diff
@@ -571,6 +571,74 @@ def resolve_ymd(self, yearfirst, dayfirst):
         return year, month, day
 
 
+def next_jump(tokens, idx, skip_comma=True):
+    """
+    Get the next sibling and its index, skipping over whitespace
+    and (if specified) commas.
+
+    Parameters
```

Comment:
This is inconsistent with the rest of the documentation, where we're using the standard sphinx style. I'm not opposed to the idea of converting over the documentation to using this numpy-style, but I don't know if I want it to be a mishmash, which confuses new contributors.

Reply:
That's fair, will try to convert (I'm much less familiar with the sphinx style, so will bear review).  Other than "I like it better" Im not sure if there is a compelling reason to convert to the numpy-style.  Regardless, that would be its own Issue.